metrics_utility/automation_controller_billing/helpers.py

```diff
@@ -1,9 +1,122 @@
 import json
 
 from itertools import chain
+from typing import Any, Dict, Optional, Tuple
 
 import pandas as pd
 
+from django.db import connection
+from django.utils.dateparse import parse_datetime
+
+from metrics_utility.logger import logger
+
+
+def get_last_entries_from_db() -> Optional[str]:
+    """
+    Get AUTOMATION_ANALYTICS_LAST_ENTRIES directly from database.
+
+    Returns:
+        Optional[str]: JSON string from database, or None if not found or error occurs
+    """
+    try:
+        with connection.cursor() as cursor:
+            cursor.execute("""
+                SELECT value
+                FROM conf_setting
+                WHERE key = 'AUTOMATION_ANALYTICS_LAST_ENTRIES'
+                LIMIT 1
+            """)
+            result = cursor.fetchone()
+
+            if result and result[0]:
+                return json.loads(result[0], object_hook=datetime_hook)  # This is the JSON value
+    except Exception as e:
+        logger.error(f'Error getting AUTOMATION_ANALYTICS_LAST_ENTRIES from database: {e}')
+    return None
```

Comment:
```suggestion
    return {}
```

(previously, that `last_entries.value if last_entries is not None else '') or '{}'` ensured that `json_loads('{}')` would happen even when nothing is found .. so we should return an empty dict, instead of None, last entries wasn't in the db)